spark/init.sh

```diff
@@ -24,119 +24,80 @@ then
 
 # Pre-packaged spark version:
 else 
-  case "$SPARK_VERSION" in
-    0.7.3)
+ case "$SPARK_VERSION" in
+    # 0.7.3 - 1.0.2
+    0\.[7-9]\.[0-3]|1\.0\.[0-2])
```

Comment:
I'm not sure this will work as the 0.8.0 and 0.9.0 have incubating in their package names. My take would be be to keep the existing long form for these early versions